src/components/ArticlesByTopic.jsx

```diff
@@ -0,0 +1,16 @@
+import { useParams } from "react-router-dom";
```

Comment:
All articles and articles by topic should be that same component as UI looks the same. You can just add some logical so that the topic title is only displayed if topics = something. 🙂